AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Wasn't it 7 conversions out of 100?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Don't get the burger joke.</p>

Reply:
<blockquote>Wasn't it 7 conversions out of 100?</blockquote><p>Ha ha indeed, the results changed from the previous run, I'll update this</p><p> </p><blockquote>Don't get the burger joke.</blockquote><p>Yeah, neither of you get it --&gt; back to the joke drawing board ;)</p>